litestar/dto/base_dto.py

```diff
@@ -195,11 +195,31 @@ def create_openapi_schema(
     ) -> Reference | Schema:
         """Create an OpenAPI request body.
 
+        Args:
+            field_definition: A parsed type annotation that represents the annotation used on the handler.
+            handler_id: ID of the route handler for which to create a DTO instance.
+            schema_creator: A factory for creating schemas. Has a ``for_field_definition()`` method that accepts a
```

Comment:
This weird wording is because `SchemaFactory` is an internal type, but its exposed on a public interface here. We should probably replace the annotation with a Protocol.